.config/notice-file/README.md

```diff
@@ -0,0 +1,28 @@
+# Notice.txt File Configuration
+
+We are automatically generating Notice.txt by using first-level dependencies of the project. The related pipeline uses `config.yaml` stored in this folder.
+
+
+## Configuration
+
+Sample:
+
+```
+title: "Mattermost Playbooks"
```

Comment:
Mattermost Playbooks? It should be Mattermost Boards Plugin right?

Reply:
This is the "sample" configuration from the notice generator. It's debatable if we need to repeatedly document how the tool works, but for now I've kept it consistent with other plugins.